config/config.go

```diff
@@ -93,6 +93,10 @@ const (
 	FsDriverFscache string = "fscache"
 )
 
+type Experimental struct {
```

Comment:
Could we add an example in `toml` and related document?

Reply:
Good point, already changed the example toml configuraion file